custom_components/unifiprotect/config_flow.py

```diff
@@ -72,6 +73,7 @@ async def _async_get_nvr_data(
         if CONF_HOST in user_input:
             host = user_input[CONF_HOST]
             port = user_input.get(CONF_PORT, DEFAULT_PORT)
+            verify_ssl = user_input.get(CONF_VERIFY_SSL, True)
```

Comment:
```suggestion
            verify_ssl = user_input.get(CONF_VERIFY_SSL, DEFAULT_VERIFY_SSL)
```

Most people won't have a valid ssl so we probably shouldn't check it by default since users won't know what to do to get it to work

 `DEFAULT_VERIFY_SSL` to `False`

Reply:
I figured you were going to say that. I am really torn on that, but I cannot argue with you that most people will have invalid certs, but it is still a "bad practice" to default to false and the Devops in me hates it.

Reply:
Since installing a valid ssl requires user interaction with the device, we should hope they are also proficient enough to check the box 🤣